todo_app/app.py

```diff
@@ -1,24 +1,33 @@
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-from todo_app.data.session_items import *
-
+from todo_app.data.session_items import addCard, getLists, getCards, moveCard, removeCard
 
 app = Flask(__name__)
 app.config.from_object(Config)
  
 
-@app.route('/', methods=['GET'])
+@app.route('/')
 def index():
-    items = get_items()
-    return render_template('index.html', items = items)
+    return render_template('index.html', lists=getLists(),cards=getCards())
+
+
+@app.route('/',methods=['POST'])
+def test():
+    name=request.form.get("card_name")
+    addCard(name,request.form['submit_button'])
+    return redirect('/')
+
+
+@app.route('/move/<card_id>/<list_id>',methods=['GET','POST'])
```

Comment:
Ideally the move and delete endpoints would use POST requests and the html use `<form>`s and `<button>`s instead of anchor tags, but it's not a big deal